core/store/migrations/migration1568390387/migrate.go

```diff
@@ -1,18 +1,33 @@
 package migration1568390387
 
 import (
+	"time"
+
 	"github.com/jinzhu/gorm"
 	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/store/assets"
 )
 
+type Encumbrance struct {
+	ID                     uint         `gorm:"primary_key;auto_increment"`
+	Payment                *assets.Link `gorm:"type:varchar(255)"`
```

Comment:
I believe this will accept a numeric type, should we use this to optimize on disk space?

Reply:
Possibly, but that seems outside the scope of this bug fix. Added a story: https://www.pivotaltracker.com/story/show/169638029